pkg/controller/operator.go

```diff
@@ -157,3 +167,24 @@ func (s *Support) Run(ctx context.Context, controller *controllercmd.ControllerC
 	<-ctx.Done()
 	return nil
 }
+
+func isRunning(ctx context.Context, config *rest.Config) wait.ConditionFunc {
+	return func() (bool, error) {
+		c, err := corev1client.NewForConfig(config)
+		if err != nil {
+			return false, err
+		}
+		pod, err := c.Pods(os.Getenv("POD_NAMESPACE")).Get(ctx, os.Getenv("POD_NAME"), metav1.GetOptions{})
+		if err != nil {
+			return false, err
```

Comment:
Actually this is questionable and very likely wrong, because if we return err then the `wait.PollImmediate` won't try again.